src/django/api/countries.py

```diff
@@ -241,7 +241,7 @@
     ('UZ', 'Uzbekistan'),
     ('VU', 'Vanuatu'),
     ('VE', 'Venezuela, Bolivarian Republic of'),
-    ('VN', 'Viet Nam'),
+    ('VN', 'Vietnam'),
```

Comment:
Is this going to prevent parsing facility lists that have `Viet Nam` listed as the country name? That seems to be the ISO standard for writing the country name.

Reply:
No, parsing should still work correctly (and for both variants) because they are covered further down in the file:

https://github.com/open-apparel-registry/open-apparel-registry/blob/e7e8177aafebde721fdcbd6b8ac9b1e99e488acf/src/django/api/countries.py#L561-L562